### PR TITLE
moving rest api

### PR DESCRIPTION
**Pull request by daylerVMT**
add rest-api dayler

### CODE REVIEW

File: CONVERT_SERVICES/src/converter/controller/rest_api_dayler/resources/file_downloader.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        file = send_file(
+            path_or_file=filepath,
+            mimetype="application/octet-stream",
+            as_attachment=True,
+            attachment_filename=name
+        )
+        return file
+
+    def delete(self, file_id):
+        pass
```
Comment (FreddyClaros) [inline CONVERT_SERVICES/src/converter/controller/rest_api_dayler/resources/file_downloader.py:19]: here if you can put and additional line, just for conventions, please

File: CONVERT_SERVICES/src/converter/controller/rest_api_dayler/resources/file_uploader.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        parse.add_argument('name', type=str)
+        args = parse.parse_args()
+        image_file = args['file']
+        name = args['name']
+        filename = f"{name}.jpg"
+        image_file.save(f"files/{filename}")
+        return {"file_id": filename}
+
+    def get(self):
+        pass
```
Comment (FreddyClaros) [inline CONVERT_SERVICES/src/converter/controller/rest_api_dayler/resources/file_uploader.py:18]: add additional line here too please

File: CONVERT_SERVICES/src/converter/controller/rest_api_dayler/main.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from flask import Flask
```
Comment (FreddyClaros) [inline CONVERT_SERVICES/src/converter/controller/rest_api_dayler/main.py:1]: all seems ok to me, but you're forgetting the header for .py archives, add them please

File: CONVERT_SERVICES/src/converter/controller/rest_api_dayler/resources/file_downloader.py (hunk 1 above)
Comment (daylerVMT) [reply CONVERT_SERVICES/src/converter/controller/rest_api_dayler/resources/file_downloader.py:19]: okey

**Review by abrahamUrquiola — APPROVED** [review]

**Review by FreddyClaros — APPROVED** [review]
looks good  and corrections made, good job I approve

**Pull request merged**